setup.py

```diff
@@ -69,10 +69,12 @@ def get_adccore_data():
 
     # TODO: find a more sustainable and clean solution ASAP
     if is_conda_build():
-        os.environ["LDFLAGS_LD"] = \
-            os.environ["LDFLAGS_LD"].replace("-dead_strip_dylibs", "")
-        os.environ["LDFLAGS"] = \
-            os.environ["LDFLAGS"].replace("-Wl,-dead_strip_dylibs", "")
+        clean_flags = ["LDFLAGS_LD", "LDFLAGS"]
+        for flag in clean_flags:
```

Comment:
Merge these lines and make the list a tuple (faster).

Reply:
Done.